### PR TITLE
made a seprate Active and Completed task file

### PR DESCRIPTION
**Pull request by aman7870**
Hi @whyDontI, i have separated the files Active and Completed. Please review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "pinst": "^2.1.6"
   },
   "engines": {
-    "node": "14.x"
+    "node": "10.19.0"
```
Comment (whyDontI) [inline package.json:54]: Let's not commit changes that are not related to the issue.
Can we revert this change?

**Review by whyDontI — CHANGES_REQUESTED** [review]

**Comment by aman7870** [comment]
the issue has been already been resolved. So, closing this PR
